app/tests/components/MessageBox.js

```diff
@@ -0,0 +1,37 @@
+import React from '../../../node_modules/react/addons';
+import MessageBox from '../../components/messageBox/index.jsx';
+import {expect} from 'chai';
+import {describe, it} from 'mocha';
+
+const {renderIntoDocument, scryRenderedDOMComponentsWithTag, Simulate} = React.addons.TestUtils;
+
+console.log('ms', MessageBox)
+
+describe('MessageBox', () => {
+
+    // ...
+
+    it('invokes callback when a button is clicked', () => {
```

Comment:
Is this really what this test tests?
Looking at the test it looks like the more appropriate name would be 'clears message box after sending message'.
Also we could add more tests later on, to test other aspects of this component.